yt/yt/core/concurrency/fair_share_weighted_thread_pool.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "public.h"
+
+#include <yt/yt/core/actions/invoker.h>
+#include <yt/yt/core/actions/public.h>
+
+namespace NYT::NConcurrency {
+
+////////////////////////////////////////////////////////////////////////////////
+
+struct IInvokerWithExpectedBytes
```

Comment:
this is the core component, the word "bytes" is clearly indicated here, as it seems to me that in the last review I suggested naming this class differently and making the interface more universal.

for example IWeightedInvoker

Reply:
>weight is something different. Weight is assigned to buckets and represents importance or share of resources allocated
>
>Here bytes is something consumable which represents size of the job, or cost of the operation

Reply:
but this does not answer the question, why should the core component know about bytes? Maybe it's the size? Maybe it's cost?